src/screens/LoginScreen/index.js

```diff
@@ -1,26 +1,50 @@
 import React, { useCallback, useContext } from "react";
 import { withRouter, Redirect } from "react-router";
 import { AuthContext } from "../../Auth.js";
-import { Login } from "./styles";
 import app from "../../firebase";
+import LoginForm from "./components/LoginForm";
+import formFields from "./formFields";
+import { Formik } from "formik";
+import { Login, Container } from "./styles";
 
 const LoginScreen = ({ history }) => {
   const handleLogin = useCallback(
-    async event => {
-      event.preventDefault();
-      const { email, password } = event.target.elements;
+    async (
+      {
+        email,
+        password,
+      },
+      { setSubmitting }
+    ) => {
       try {
         await app
           .auth()
-          .signInWithEmailAndPassword(email.value, password.value);
+          .signInWithEmailAndPassword(email, password);
+        setSubmitting(false);
         history.push("/");
       } catch (error) {
-        alert(error);
+        console.log(error);
       }
     },
     [history]
   );
 
+  const validateForm = useCallback(({ email, password }) => {
```

Comment:
side note: we need to figure out a way to display the errors that we get back from the server (e.g. incorrect password) Right now we only log to console

Reply:
we can generate a log file whenever errors occur.